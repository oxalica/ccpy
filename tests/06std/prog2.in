def main():
  s = 0
  for i in range(100):
    s = s + i
  print(s)

main()
