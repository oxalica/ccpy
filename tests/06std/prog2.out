Std done

4950
Done
